scripts/userstyles.yml

```diff
@@ -424,14 +424,6 @@ userstyles:
             One solution might be changing the theme to either light or dark via **Preferences** > **Appearance** > **Site theme**.
       current-maintainers: [*isabelroses]
       past-maintainers: [*andreasgrafen]
-  modrinth:
-    name: Modrinth
-    category: game
-    icon: modrinth
-    color: green
-    readme:
-      app-link: "https://modrinth.com"
-      current-maintainers: [*thismoon]
```

Comment:
I assume you didn't mean to remove this?

Reply:
Oops yeah lol. I think it was to fix the alphabetical order but that doesn't really matter ahah.